### ISSUE TITLE
saving session

### ISSUE THREAD

**Issue by ghost** [jimeh/tmuxifier#9]
Is there a way to save the current state of a session?

**Comment by leesei** [comment]
I guess we can achieve this by parsing tmux's status:

```sh
tmux ls # to list sessions
tmux list-windows -t <session> # to list windows and layouts
```

**Comment by jimeh** [comment]
@leesei Indeed, I haven't really spent any time on it as it's a bit complicated, and ideally I'd like to output the resulting Tmuxifier layout in a humanly editable format. But at the same time there would probably need to be a separation between manually created and saved sessions, as saved ones would obviously lack the ability of running commands as part of setting up the layout.

**Comment by leesei** [comment]
I think we can include a script in the package or wiki to help user generate a session **template** in Tmuxifier layout format.
It is expected for the user to customize and review the layout before moving it to the layout folder.
